Makefile

```diff
@@ -28,7 +28,7 @@ CT_NODE_NAME ?= 'test@127.0.0.1'
 CT_READABLE ?= true
 CT_COVER_EXPORT_PREFIX ?= $(PROFILE)
 
-export REBAR_GIT_CLONE_OPTIONS += --depth=1
+export REBAR_GIT_CLONE_OPTIONS += --depth=3
```

Comment:
why this change ?

Reply:
Oops, accidentally put local only changes into a commit. Thanks for pointing out.